Bikes/charts/bikes/values.yaml

```diff
@@ -20,15 +26,38 @@ ingress:
   annotations:
     kubernetes.io/ingress.class: addon-http-application-routing
   path: /
+  # hosts:
+  #   - chart-example.local
   tls: []
-  
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
 secrets:
   mongo:
     collection: bikes 
-    connectionString: mongodb://databases-mongo
-
+    connectionstring: mongodb://databases-mongo
+  # Optionally specify a set of secret objects whose values
```

Comment:
I chose to keep the comments because I thought that if users were looking to this file (so likely trying to understand something), they would be happy to find more information about what these values are.
That said if you guys have strong opinions about that, I don't mind changing it but I really think this might help some of our users.

---
In reply to: [266656047](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/20#discussion_r266656047) [](ancestors = 266656047)

Reply:
Cool, this makes sense to me. I think it's probably fine if we leave it.

Reply:
Alright great, thanks Yamini!

---
In reply to: [267030870](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/20#discussion_r267030870) [](ancestors = 267030870)